cloudvolume/meshservicegraphene.py

```diff
@@ -65,8 +67,29 @@ def decode_mesh_buffer(fragment):
     'num_vertices': num_vertices,
     'vertices': vertices.reshape( num_vertices, 3 ),
     'faces': faces,
+    'was_draco_encoded': False
   }
 
+def decode_draco_mesh_buffer(fragment):
+    try:
+        mesh_object = DracoPy.decode_buffer_to_mesh(fragment)
+        vertices = np.array(mesh_object.points)
+        faces = np.array(mesh_object.faces)
+    except ValueError:
+        raise ValueError("Not a valid draco mesh")
+
+    assert len(vertices) % 3 == 0, "Draco mesh vertices not 3-D"
+    num_vertices = len(vertices) // 3
+
+    # For now, just return this dict until we figure out
+    # how exactly to deal with Draco's lossiness/duplicate vertices
+    return {
+        'num_vertices': num_vertices,
+        'vertices': vertices.reshape(num_vertices, 3),
+        'faces': faces,
+        'was_draco_encoded': True
```

Comment:
Maybe it would be better to have 'encoding': 'draco' | 'precomputed' in case we'd like to use more encodings.

Reply:
Agreed. I will fix this in the next commit